File: Retrospective.Data.Model/Model/Team.cs
using System;
using MongoDB.Bson;

namespace Retrospective.Data.Model
{
    public class Team
    {
        public ObjectId? Id {get;set;}
        public string Name {get; set;}
        public TeamMember[] Members {get;set;}

        public Invitation[] Invited {get;set;}
        

    }
}

File: web/client/src/app/models/team.ts
import { Meeting } from "./meeting";
import { TeamMember } from "./teammember";
import { Invitation } from "./invitation";

export class Team {
  teamId: string;
  name: string;
  owner: string
  members: TeamMember[];
  invited: Invitation[];
  meetings: Meeting[];
  message: string;
}